app/controllers/concerns/hyrax/collections_controller_behavior_decorator.rb

```diff
@@ -10,6 +10,13 @@ def load_member_subcollections
 
       @subcollection_docs.sort_by! { |doc| doc.title.first&.downcase }
     end
+
+    def show
+      # OVERRIDE Hyrax 3.6.0 to default sort by title instead of relevance
+      params[:sort] ||= "title_ssi asc"
```

Comment:
what do you think about something like this?

```suggestion
      params[:sort] ||= "#{Hyrax::CollectionSearchBuilderDecorator::SORT_FIELD} asc"
```